mlrun/model_monitoring/db/tsdb/timescaledb/timescaledb_connector.py

```diff
@@ -289,6 +289,60 @@ def _enrich_mep_with_last_request(
     def read_predictions(self, *args, **kwargs):
         return self._predictions_queries.read_predictions(*args, **kwargs)
 
+    def _get_records(
+        self,
+        table: str,
+        start: datetime.datetime,
+        end: datetime.datetime,
+        endpoint_id: Optional[str] = None,
+        columns: Optional[list[str]] = None,
+    ) -> pd.DataFrame:
+        """
+        Get raw records from TimescaleDB as pandas DataFrame (similar to TDEngine).
```

Comment:
since we are going to remove TDEngine, I prefer if we don't mention it in TimeScaleDB